appveyor.yml

```diff
@@ -54,21 +54,12 @@ install:
     - IF %dependencies%==highest appveyor-retry composer update --no-suggest --ignore-platform-reqs
     - composer show
     # Install XDebug for code coverage
-    - ps: |
-        $client = New-Object System.Net.WebClient
-        $xdebugUrl = "https://xdebug.org/files/php_xdebug-$env:xdebug_ver-nts-x86_64.dll"
-        $phpDir = (Get-Item (Get-Command php).Source).Directory.FullName
-        $xdebugPath = Join-Path $phpDir ext\xdebug.dll
-        $client.DownloadFile($xdebugUrl, $xdebugPath)
-        $phpIniPath = Join-Path $phpDir php.ini
-        Add-Content $phpIniPath @"
-        zend_extension=$xdebugPath
-        "@
+    - ps: if ($env:PHP -eq "1" ) { .ci\appveyor_install_xdebug.ps1 }
```

Comment:
could you please explain what is it for, the main idea of this PR? I'm not sure I follow it

Reply:
We cache php, but before this change we applied xdebug on every single run, meaning we would have duplicate php ini entries for the xdebug configuration, which crashes phpspec.

The reason i moved it to a new file was due to strange windows errors with the`@"<text>"@` notation

Reply:
Could it be like this?
```diff
-        Add-Content $phpIniPath @"
-        zend_extension=$xdebugPath
-        "@
+       IF %PHP%==1 echo extension=php_openssl.dll >> $phpIniPath
```

Reply:
Well, if we don't have php cached we need to download xdebug, which is what this step also does.

Reply:
Then I guess that's the way to go.